src/TTextEdit.cpp

```diff
@@ -564,6 +619,7 @@ void TTextEdit::drawForeground(QPainter& painter, const QRect& r)
             break;
         }
         drawLine(p, i + lineOffset, i);
+        updateHorizontalScrollBar(i + lineOffset);
```

Comment:
I would put the error console check here and wrap it in the q_unlikely, just so the way more used main display and miniconsoles aren't paying the price here